### DISCUSSION TITLE
[SOLVED] node.js

### DISCUSSION THREAD

**Discussion by Skywriter960** [martinrotter/rssguard#639] [Q&A]
Now, I hope this is my last question:

How integrat note.js in the portable program folder rss guard.

I have a zip-file of note.js for portable use ... ?!

I cann't check adblock in the settings ... and the error message show me your documentation site ...

**Comment by martinrotter** [comment]
This depends on what version of RSS Guard you want to use.

4.1.2 - you need to install Node.js from their website, preferably with the installer. Installer will add `node.exe` and `npm.cmd` executables to your `PATH` environment variable automatically. Then you need to install needed package `@cliqz/adblocker` with NPM.

4.2.0 (devbuild) - has reworked node.js integration. Just install Node.js with installer in next-next-next style. RSS Guard will install NPM packages automatically when needed.

4.2.0 is not yet released. I plan to release in a month probably.

**Comment by Skywriter960** [comment]
Hello,

it looks like I am now out of my depth. Node.js I have installed in the program folder RSS Guard. Everything from ZIP files.

I don't know anything about environment variables on the stick.

The dialog box from the documentation for the settings AdBlock is displayed differently for me. The button 'test configuration' is missing.

What is meant by ' next-next-next style'? 

As long as the installation procedures for 'Portable' do not run automatically and need to be re-sharpened, I'm out again.

I keep reading that there are 2 RSS Guard versions. I have

RSS Guard
Version: 4.1.2 (created under Windows/x86_64)
Revision: 1f5157fd
Creation date: 10.01.22 07:26
Qt version: 5.15.2 (compiled against 5.15.2)

Do I have the wrong one? This is confusing me.

Too bad.

Greetings,
Skywriter
